packages/pf-codemods/README.md

```diff
@@ -57,3 +57,21 @@ Out:
 ```jsx
 <ToolbarContent visibility={{ default: "hidden" }} />
 ```
+
+### tooltip-remove-props [(#8231)](https://github.com/patternfly/patternfly-react/pull/8231)
+
+We've removed the `boundary`, `tippyProps`, and `isAppLauncher` properties from Tooltip.
+
+#### Examples
+
+In:
+
+```jsx
+<Tooltip boundary={} tippyProps={} isAppLauncher />
+```
+
+Out:
+
+```jsx
+<Tooltip >
```

Comment:
Should this be
```suggestion
<Tooltip  />
```

Reply:
you're right, would also probably leave 4 spaces

Reply:
I'll update. Prettier is automatically removing the extra spacing so if we want to keep the exact output we'll have to watch it because the file needs to be saved without formatting.